crates/blockchain-tree-api/src/lib.rs

```diff
@@ -198,6 +198,36 @@ impl CanonicalOutcome {
     }
 }
 
+/// Block inclusion can be valid, accepted, or invalid. Invalid blocks are returned as an error
+/// variant.
+///
+/// If we don't know the block's parent, we return `Disconnected`, as we can't claim that the block
+/// is valid or not.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum BlockStatusTwo {
```

Comment:
smol nit: can we make this a number so that we are consistent w/ `BlockchainProvider2` and it's clear that it should be renamed in the fut